ERROR   2022-05-17 22:29:07,905  C:\work\Study\pytest-auto-api2\logs\error.log:conftest.py:131 异常用例数: 0
ERROR   2022-05-17 22:29:07,905  C:\work\Study\pytest-auto-api2\logs\error.log:conftest.py:132 失败用例数: 0
